### PR TITLE
Feat/video panel: Video and description

### PR DESCRIPTION
**Pull request by trizgb**
Add to Video screen:
- VideoPanel component, with image and some basic info like username, title and subscribers, and buttons for interaction.
- VideoPanelInfo component, with description box and user info and some buttons.

Also, Lottie package was added to show animations when some buttons are clicked (like and subscribe buttons).

### CODE REVIEW

File: src/hooks/useVideo.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { useQuery } from '@tanstack/react-query'
+import { getVideoById } from 'api'
+
+const useVideo = id => {
```
Comment (abelta) [inline src/hooks/useVideo.js]: const useVideo = ({ id }) => {

Mejor mantenibilidad para poner parámetros opcionales en el futuro.

File: src/components/VideoPanelInfo/VideoPanelInfo.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    setShowFullDescription(!showFullDescription)
+  }
+
+  // const getLastWordBeforeMore = () => {
+  //   const words = video.description.split(' ')
+  //   const sliceLength = isMobile ? 100 : 500
+  //   const truncatedWords = words.slice(0, sliceLength)
+  //   const lastWord = truncatedWords[truncatedWords.length - 1]
+  //   return lastWord
+  // }
```
Comment (trizgb) [inline src/components/VideoPanelInfo/VideoPanelInfo.js]: Remove commented code

File: src/components/VideoPanelInfo/VideoPanelInfo.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+        <span style={{ margin: '10px 10px 5px 10px' }}>
+          {video.views} views
+        </span>
+        <span style={{ margin: '10px 10px 5px 10px' }}>
+          {formatDistanceToNow(video.timestamp)}
+        </span>
+      </div>
+      <div style={{ padding: '10px' }}>
+        <span style={{ margin: '10px' }}>
+          {' '}
```
Comment (trizgb) [inline src/components/VideoPanelInfo/VideoPanelInfo.js]: Remove this line

File: src/components/VideoPanelInfo/VideoPanelInfo.js [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+              {/* <span
+                style={{
+                  backgroundImage:
+                    'linear-gradient(to right, transparent 80%, transparent 100%)',
+                  display: 'inline-block',
+                }}
+              >
+                {getLastWordBeforeMore()}
+              </span>
+              ... */}
```
Comment (trizgb) [inline src/components/VideoPanelInfo/VideoPanelInfo.js]: Remove commeted code

File: src/hooks/useBreakPoint.js [hunk 5]
```diff
@@ -2,8 +2,8 @@ import { useState, useEffect } from 'react'
 
 const useBreakPoint = () => {
   const [breakPoints, setBreakPoints] = useState({
-    isMobile: false,
-    isMobileLarge: false,
+    mobile: false,
```
Comment (rodriguezmontesinoseva) [inline src/hooks/useBreakPoint.js]: Modificar mobile por isMobile y mobileLarge por isMobileLarge

**Review by Gitow93 — APPROVED** [review]

**Pull request merged**